llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -3924,38 +3924,55 @@ static SDValue lowerBuildVectorViaPacking(SDValue Op, SelectionDAG &DAG,
                      DAG.getBuildVector(WideVecVT, DL, NewOperands));
 }
 
-// Convert to an vXf16 build_vector to vXi16 with bitcasts.
-static SDValue lowerBUILD_VECTORvXf16(SDValue Op, SelectionDAG &DAG) {
```

Comment:
I don't understand the removal of this combined with the review description.  We clearly did handle some non-splat cases here?

Reply:
We only handled them if the Zfbfmin/Zfhmin were not enabled.